catroid/src/org/catrobat/catroid/devices/mindstorms/nxt/sensors/NXTSensorService.java

```diff
@@ -109,6 +112,18 @@ private NXTSensor createSensor(String sensorTypeName, int port) {
         return sensor;
 	}
 
+	public void deactivateAllSensors(MindstormsConnection connection) {
+
+		for (int port = 0; port < 4; port++) {
+
+			Command command = new Command(CommandType.DIRECT_COMMAND, CommandByte.SET_INPUT_MODE, true);
```

Comment:
if the third parameter is true, it is a reply command, so the nxt will send a reply message. But in line 124 you use send, so the reply is not handled. Please use a non reply command here. So the third parameter should be false.